CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.5.0] - 2017-12-12
+
+### Added
+- beetmoverscript support for Devedition releases
+
+### Changed
+- stop uploading checksums.asc files as `.beet`  under beetmover-checksums
+- `get_release_props` and `update_props`  functions now take context as argument
```

Comment:
This is a breaking change. It's not as critical for beetmoverscript to respect semver since we have nothing downstream. But if we follow semver, we would bump the major version. 4.0.0.